svg/painting/marker-007.svg

```diff
@@ -0,0 +1,23 @@
+<svg width="100%" height="100%" viewBox="0 0 480 360"
```

Comment:
I notice that in Firefox this `svg/painting/marker-007.svg` test displays nothing. Is that an expected failure? Gecko doesn’t conform to the spec on this?